lib/sql.js

```diff
@@ -1113,9 +1151,7 @@ SQLConnector.prototype._buildWhere = function(model, where) {
     }
     const p = props[key];
     if (p == null) {
-      // Unknown property, ignore it
-      debug('Unknown property %s is skipped for model %s', key, model);
-      continue;
+      throw new Error('Unknown property ' + key + ' used for model ' + model);
```

Comment:
Please add a feature flag allowing users to revert back to the current behavior in their existing projects. See https://github.com/strongloop/loopback-connector/pull/132#issuecomment-419354983

Reply:
Done